testsuite/libra-fuzzer/src/fuzz_targets/language_transaction_execution.rs

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::FuzzTargetImpl;
+use language_e2e_tests::account_universe::{
+    all_transactions_strategy, log_balance_strategy, run_and_assert_universe, AccountUniverseGen,
+};
+use libra_proptest_helpers::ValueGenerator;
+use proptest::{
+    collection::vec,
+    strategy::{Strategy, ValueTree},
+    test_runner::{self, TestRunner},
+};
+use rand::RngCore;
+
+#[derive(Clone, Debug, Default)]
+pub struct LanguageTransactionExecution;
+
+impl FuzzTargetImpl for LanguageTransactionExecution {
+    fn name(&self) -> &'static str {
+        module_name!()
+    }
+
+    fn description(&self) -> &'static str {
+        "Storage save blocks"
+    }
+
+    fn generate(&self, _idx: usize, _gen: &mut ValueGenerator) -> Option<Vec<u8>> {
+        let mut output = vec![0u8; 4096];
+        let mut rng = rand::thread_rng();
+        rng.fill_bytes(&mut output);
+        Some(output)
+    }
+
+    fn fuzz(&self, data: &[u8]) {
+        let passthrough_rng =
+            test_runner::TestRng::from_seed(test_runner::RngAlgorithm::PassThrough, &data);
+
+        let config = test_runner::Config::default();
+        let mut runner = TestRunner::new_with_rng(config, passthrough_rng);
+
+        let txn_trategy = vec(all_transactions_strategy(0, 1_000_000), 1..40);
+        let txn_trategy_tree = match txn_trategy.new_tree(&mut runner) {
+            Ok(x) => x,
+            Err(_) => return,
+        };
+        let txn = txn_trategy_tree.current();
+
+        let universe_strategy =
+            AccountUniverseGen::strategy(2..20, log_balance_strategy(10_000_000));
+        let universe_strategy_tree = match universe_strategy.new_tree(&mut runner) {
+            Ok(x) => x,
+            Err(_) => return,
+        };
+        let universe = universe_strategy_tree.current();
```

Comment:
Could you use `ValueGenerator` for these?

Reply:
ValueGenerator doesn't have a method to initialize with a specific RNG. Should I add this method to `ValueGenerator`?

Reply:
yeah!